applications/new_schema.py

```diff
@@ -103,11 +102,10 @@ class Input(BerthApplicationInput):
     berth_application = graphene.Field(BerthApplicationNode)
 
     @classmethod
-    @login_required
-    @superuser_required
+    @view_permission_required(CustomerProfile)
```

Comment:
you will also be able to see the `BerthLease` :derp:

Reply:
So close yet so far 😭 I even thought about it, but apparently thought it wasn't necessary.